src/components/SchemaRow.tsx

```diff
@@ -50,49 +52,55 @@ export const SchemaPropertyRow: React.FunctionComponent<Pick<ISchemaRow, 'rowOpt
 
   return (
     <>
-      {!isBrokenRef && isParentNode(treeListNode) && Tree.getLevel(treeListNode) > 0 ? (
-        <Caret
-          isExpanded={!!rowOptions.isExpanded}
-          style={{
-            width: CARET_ICON_BOX_DIMENSION,
-            height: CARET_ICON_BOX_DIMENSION,
-            ...(!isBrokenRef && Tree.getLevel(treeListNode) === 0
-              ? {
-                  position: 'relative',
-                }
-              : {
-                  left: CARET_ICON_BOX_DIMENSION * -1 + SCHEMA_ROW_OFFSET / -2,
-                }),
-          }}
-          size={CARET_ICON_SIZE}
+      <Flex my={2}>
+        {!isBrokenRef && isParentNode(treeListNode) && Tree.getLevel(treeListNode) > 0 ? (
+          <Caret
+            isExpanded={!!rowOptions.isExpanded}
+            style={{
+              width: CARET_ICON_BOX_DIMENSION,
+              height: CARET_ICON_BOX_DIMENSION,
+              ...(!isBrokenRef && Tree.getLevel(treeListNode) === 0
+                ? {
+                    position: 'relative',
+                  }
+                : {
+                    left: CARET_ICON_BOX_DIMENSION * -1 + SCHEMA_ROW_OFFSET / -2,
+                  }),
+            }}
+            size={CARET_ICON_SIZE}
+          />
+        ) : null}
+
+        {schemaNode.subpath.length > 0 &&
+          isCombiner(schemaNode.subpath[0]) &&
+          schemaNode.parent?.children?.indexOf(schemaNode as any) !== 0 && <Divider kind={schemaNode.subpath[0]} />}
+
+        <Flex flex={1} className="truncate">
+          <Property onGoToRef={onGoToRef} />
+          <Format />
+        </Flex>
+        <Properties
+          required={isPropertyRequired(schemaNode)}
+          deprecated={isRegularNode(schemaNode) && schemaNode.deprecated}
+          validations={
+            isRegularNode(schemaNode)
+              ? schemaNode.validations
+              : {}
+          }
         />
-      ) : null}
-
-      {schemaNode.subpath.length > 0 &&
-        isCombiner(schemaNode.subpath[0]) &&
-        schemaNode.parent?.children?.indexOf(schemaNode as any) !== 0 && <Divider kind={schemaNode.subpath[0]} />}
-
-      <div className="flex-1 flex truncate">
-        <Property onGoToRef={onGoToRef} />
-        <Format />
-        {typeof description === 'string' && description.length > 0 && <Description value={description} />}
-      </div>
-
-      <Validations
-        required={isPropertyRequired(schemaNode)}
-        deprecated={isRegularNode(schemaNode) && schemaNode.deprecated}
-        validations={
+      </Flex>
+
+      {typeof description === 'string' && description.length > 0 && (
+        <Flex flex={1} my={2} className="truncate">
```

Comment:
`<Flex textOverflow="truncate"></Flex>`

Reply:
Thanks, I didn't know this property here, I wanted to use `sl-truncate` class